java/examples/src/java/org/apache/orc/examples/CompressionWriter.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
```

Comment:
Ur, do we have this in ASF license header in the other files?

Reply:
Got it. It seems to be copied from `Driver.java`. Never mind.

Reply:
Yes, we now have license checks. The build will fail if no license exist.